docs/guide/contributing.rst

```diff
@@ -1,73 +1,203 @@
+2.  Go to a directory for storing the cloned repository. 
 
-What we expect from you, and what you can expect from us
---------------------------------------------------------
+3.  Type ``git clone``, followed by a space,
+    and then paste the project URL:
 
-We encourage contributions from experienced folks and from newcomers, whether you're new to documentation, to software, to GitHub, or to Write the Docs. We expect you to respect the effort that's gone into whatever the current state of the guide might be, and we'll treat your contributions with respect in turn.
+       ::
 
-Keep your tone friendly and encouraging. It's a good way to write docs in general, so practice with docs about docs!
+          git clone https://github.com/myname/www.git
 
-Link liberally to resources that you find useful, but make sure to provide appropriate attribution. And make sure to add something about how or why you find them useful.
+    git clone copies files from GitHub to a folder named ``www`` on your computer.
 
-For anyone new to GitHub and Git
---------------------------------
+4.  Go to the ``www`` directory:
 
-If you've never worked with Git and GitHub before, this resource is great for getting started: `How to Contribute to an Open Source Project on GitHub <https://app.egghead.io/playlists/how-to-contribute-to-an-open-source-project-on-github>`_.
+       ::
 
-To suggest small changes, you can `submit issues <https://github.com/writethedocs/www/issues>`_ via the GitHub interface. 
+          cd www
 
-For large changes, you'll probably want to learn how to fork and branch, how to work locally (although you can do most things in the GitHub interface in your branch), and how to keep your fork in sync with the main repository.
+5.  Create a new branch:
 
-If you really don't want to deal with GitHub at all, we'll take attachments or inline text emailed to guide@writethedocs.org. But we'd like to encourage you to learn some of the basic tools that we use.
+       ::
 
-Markup
-------
+          git branch branch-name
 
-Some of the current files in the guide are written in `Markdown </guide/writing/markdown/>`_, and some are written in `reStructured Text (rST) </guide/writing/reStructuredText/>`_. Either one is acceptable, although we'd prefer rST.
+    Replace ``branch-name`` with a brief description of your proposed changes. 
+    Use dashes between words. For example: ``git branch important-typo-fix``.
+
+6. Switch to the new branch:
+
+       ::
+
+          git checkout branch-name
+
+**In any text editor like `Sublime Text`_ or `Visual Studio Code`_:**
+
+1. Open the file you want to edit.
+
+2. Edit and save the file.
+
+**In a terminal window:**
```

Comment:
Would it be useful to break this up into three sections? Get the files, make the changes, send the changes back to WTD?

Reply:
I agree that the doc could still use better organization and will take another pass. However, I think we still should keep some organization of where the actions are being taken since it helped replace almost 10 occurrences of the phrase "In a terminal window...." and seemed clearer for me to read overall.

Reply:
Oh yeah, not suggesting removing those :+1: 

Happy to merge and iterate another time.